client/packages/system/src/Stock/Components/InventoryAdjustment/InventoryAdjustmentModal.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC } from 'react';
+import {
+  TextWithLabelRow,
+  useTranslation,
+  Box,
+  NumericTextInput,
+  DialogButton,
+  useNotification,
+  AdjustmentTypeInput,
+  useDialog,
+} from '@openmsupply-client/common';
+import { StockLineRowFragment, useInventoryAdjustment } from '../../api';
+import { InventoryAdjustmentReasonSearchInput, usePackVariant } from '../../..';
+import { InventoryAdjustmentDirectionInput } from './InventoryAdjustmentDirectionSearchInput';
+import { INPUT_WIDTH, StyledInputRow } from '../StyledInputRow';
+
+interface InventoryAdjustmentModalProps {
+  stockLine: StockLineRowFragment;
+  isOpen: boolean;
+  onClose: () => void;
+}
+export const InventoryAdjustmentModal: FC<InventoryAdjustmentModalProps> = ({
+  stockLine,
+  isOpen,
+  onClose,
+}) => {
+  const t = useTranslation('inventory');
+  const { success } = useNotification();
+  const { Modal } = useDialog({ isOpen, onClose });
+
+  const { draft, setDraft, create } = useInventoryAdjustment(stockLine);
+
+  const { asPackVariant } = usePackVariant(
+    stockLine.itemId,
+    stockLine.item.unitName ?? null
+  );
+  const packUnit = asPackVariant(stockLine.packSize);
+
+  const saveDisabled = draft.adjustment === 0;
+
+  const save = async () => {
+    try {
+      await create();
+      const successSnack = success(t('messages.inventory-adjustment-saved'));
+      successSnack();
+      onClose();
+    } catch {
+      // TODO: handle error if no reason selected when reasons required
+    }
+  };
+
+  return (
+    <Modal
+      sx={{ maxWidth: 'unset', minWidth: 700, minHeight: 575 }}
+      slideAnimation={false}
+      title={t('title.stock-line-details')}
+      okButton={
+        <DialogButton variant="ok" disabled={saveDisabled} onClick={save} />
+      }
+      cancelButton={<DialogButton variant="cancel" onClick={onClose} />}
+    >
+      <Box display="flex">
```

Comment:
Ugh, this is annoying -- the form is actually centred correctly, it just looks off because all the columns are right justified. (If there was a long enough label on the left it would look "correct"). 

I've added a bit more padding on the right side only (and slightly increased the gap between columns), which helps, but it's a bit of a hack. 

68ff74c06

Reply:
Argh yeah sorry about that... would an alternative solution be to narrow the modal? There aren't other tabs constraining us to this width anymore 👀